.github/workflows/test.yml

```diff
@@ -48,12 +48,12 @@ jobs:
       with:
         python-version: ${{ matrix.python }}
 
-    - name: Install tox
+    - name: Install pkgs
       run: |
         python -m pip install --upgrade pip
-        pip install tox
+        python -m pip install -e .[testing]
```

Comment:
In line with https://github.com/gabrielcnr/pytest-datadir/pull/70/files#r1365906429, we really just need `tox` here: `tox` will then be responsible for installing the pre-existing package and any required dependencies.

Reply:
done